### PR TITLE
added submitted task modules to dashboard

### PR DESCRIPTION
**Pull request by aginzberg**
per spamgirls request

### CODE REVIEW

File: static/js/dashboard/controllers/dashboard.controller.js [hunk 1]
```diff
@@ -63,6 +66,29 @@
...
       }
 
+      function processSubmitted() {
+        self.submittedModules = {};
+        for(var i = 0; i < self.submittedTaskWorkers.length; i++) {
+          var module_id = self.submittedTaskWorkers[i].module.id;
+          if(module_id in self.submittedModules) {
+            self.submittedModules[module_id].tasks_completed += 1;
+            if(self.submittedModules[module_id].last_submission < self.submittedTaskWorkers[i].last_updated) {
+              console.log("hi");
```
Comment (neilthemathguy) [inline static/js/dashboard/controllers/dashboard.controller.js]: Take this out.
console.log("hi");

File: static/js/dashboard/controllers/dashboard.controller.js [hunk 2]
```diff
@@ -55,7 +55,10 @@
             self.acceptedTaskWorkers = data[0]['Accepted'];
             self.rejectedTaskWorkers = data[0]['Rejected'];
             self.returnedTaskWorkers = data[0]['Returned'];
+            self.submittedTaskWorkers = data[0]['Submitted'];
+            //Eventually I think we will probably want to do this processing on the backend
```
Comment (neilthemathguy) [inline static/js/dashboard/controllers/dashboard.controller.js]: let's make a log of it so that we don't forget.

**Comment by neilthemathguy** [comment]
lgtm

**Comment by dmorina** [comment]
lgtm

**Pull request merged**